aiaccel/hpo/samplers/nelder_mead_sampler.py

```diff
@@ -56,13 +56,13 @@ def before_trial(self, study: Study, trial: FrozenTrial) -> None:
             params = np.array([fixed_params[name] for name in self._search_space])
         else:
             try:
-                while True:
-                    params = self.nm.get_vertex()
-
-                    if all(low < x < high for x, (low, high) in zip(params, self._search_space.values(), strict=False)):
-                        break
-
-                    self.nm.put_value(params, np.inf)
+                params = self.nm.get_vertex()
+                params = np.array(
+                    [
+                        (high - low) * value + low
+                        for value, (low, high) in zip(params, self._search_space.values(), strict=False)
+                    ]
+                )
```

Comment:
こう書いた方が読みやすいように思います。
```suggestion
                iter_ = zip(self.nm.get_vertex(), self._search_space.values(), strict=False)
                params = np.array([(high - low) * value + low for value, (low, high) in iter_])
```